examples/advanced_circuits_algorithms/Deutsch_Jozsa_Algorithm/Deutsch Jozsa Algorithm.ipynb

```diff
@@ -0,0 +1,1037 @@
+{
```

Comment:
<p>To build the algorithm, we first prepare our qubits according to our input; the input state prepares n-qubits in the&nbsp;|0⟩ state for the query register and an extra qubit for the answer register is prepared in the&nbsp;|1⟩ state.</p><p> </p><p>superposition of all possible binary values and the...</p><p> </p><p>for the oracle&nbsp;U_f with the inputs being&nbsp;∣x,y⟩→∣x,y⊕f(x)⟩, giving....</p><p> </p><p>We apply Hadamard gates to the query register to remove the qubits from their superposition state.</p><p> </p><p>For the two possible cases, if the function is constant, then the measurement will observe all 0’s for the first qubit register and if the function is balanced, then you will observe an equal mix of 1's and 0's for the first register. However, if you have both types of measurements present for a function, the function is neither constant or balanced. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/177/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>